contrib/signet/miner

```diff
@@ -103,8 +103,9 @@ def finish_block(block, signet_solution, grind_cmd):
         block.solve()
     else:
         headhex = CBlockHeader.serialize(block).hex()
+        logging.debug('grinding headhex: {}'.format(headhex))
```

Comment:
Sorry style nit here and later on, should use `"` not `'` for strings and `%` formatting for logging
```suggestion
        logging.debug("grinding headhex: %s", headhex)
```